uring/jvm/src/main/scala/fs2/io/uring/net/UringDatagramSocketGroup.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2022 Arman Bilge
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package fs2.io.uring.net
+
+import cats.effect.kernel.Resource
+
+import com.comcast.ip4s._
+
+import fs2.io.net.DatagramSocketGroup
+import fs2.io.net.DatagramSocket
+import fs2.io.net._
+
+import java.net.ProtocolFamily
+
+private final class UringDatagramSocketGroup[F[_]] extends DatagramSocketGroup[F] {
```

Comment:
I guess we can work on this in a follow-up PR :)

Reply:
> If you don't mind one last request: can we make an effort to deduplicate some of code and share between JVM/Native?

Great idea! I'll handle it :)

> But ... I think we should try hard to share the tests, so we are running identical test suites on both platforms. We're not doing anything JVM-specific, hopefully?

I think we aren't, so it should be easy to do.